src/bot.py

```diff
@@ -42,10 +42,7 @@ async def get_contributor(self) -> Contributor | None:
 
             for repo in repos:
                 for page in range(1, 100):
-                    api = (
-                        f"https://api.github.com/repos/{org_name}/{repo}/issues"
-                        + f"?state=all&per_page=100&page={page}"
-                    )
+                    api = f"https://api.github.com/repos/{org_name}/{repo}/issues" + f"?state=all&per_page=100&page={page}"
```

Comment:
Why not make this into this to make it less confusing:

```py
api = f"https://api.github.com/repos/{org_name}/{repo}/issues?state=all&per_page=100&page={page}"
```

Reply:
@Omar8345 Noted, will fix this in an upcoming PR.